### PR TITLE
add execution permission to scripts

### PR DESCRIPTION
**Pull request by pravi**
these are shell scripts without execution permissions, adding execution permissions.

### CODE REVIEW

**Comment by jhass** [comment]
There's no need for them to be executable, https://github.com/diaspora/diaspora/blob/develop/script/install.sh#L28

However do they even still work? What's the state here, @Raven24 ?

**Comment by pravi** [comment]
lintian gives script-not-executable since it has a shebang line at start. If it doesn't hurt, it could be made executable or if that is required can we remove the shebangs?

**Comment by jhass** [comment]
I think we can remove the hashbangs, @Raven24 needs to confirm. Or remove them altogether.

**Comment by Raven24** [comment]
I just tried and it seems they should still run. They don't necessarily need to be executable, since the referenced files are never run directly, but insted `eval`d into the main script context and contain only function definitions and no directly runnable code on their own. So feel free to remove the `#!`.

I've always found those scripts to be quite the ugly hack. also that code being bash, I don't really feel confident to assess any part of their quality (even though I wrote them).
I'd rather point people to [the Ruby version](https://github.com/Raven24/diaspora-install) I wrote about a year ago, instead of the bash mess that hasn't really been touched in two years...

**Comment by jhass** [comment]
Can we get a clear statement please? Do you think we should drop them?

**Comment by Raven24** [comment]
tl;dr
if we should, for some reason keep them, drop the `#!`
but please let's remove them completely ;)

**Comment by jhass** [comment]
Okay, thanks.

@pravi do you want to update the PR to drop them or should I do it?

**Comment by svbergerem** [comment]
I guess removing the install script would fix #4218, #4275 and #4750.

**Comment by pravi** [comment]
I have updated the PR to drop them

**Comment by jhass** [comment]
Thanks!

**Comment by goobertron** [comment]
Thanks, all, for this work.

I'm guessing we should delete the [install script page](https://wiki.diasporafoundation.org/Installation/Script) on the wiki, and the link to it from the development setup section of the [main installation page](https://wiki.diasporafoundation.org/Installation) (and any other links to it on the wiki). I won't do this myself in case I'm mistaken! Apologies if someone was just about to do that; I just saw this PR and wanted to comment before I forget.
